### DISCUSSION TITLE
Upgrade Heroku installation?

### DISCUSSION THREAD

**Discussion by paulmsmith** [nocodb/nocodb#415] [Q&A]
Is there a way to quickly update an instance of NocoDB I installed via the 'Deploy to heroku button'? I can't seem to find documentation on upgrading.

**Comment by o1lab** [comment]
Possible to check this in heroku dashboard on how to point it to latest docker container ?

**Comment by tatsu38** [comment]
Hasura tells you to use the CLI. I applied it to NocoDB and the upgrade was successful.

```
git clone https://github.com/nocodb/nocodb-seed-heroku
cd nocodb-seed-heroku
heroku git:remote -a <Heroku App Name>
heroku stack:set container -a <Heroku App Name>
git push heroku main
```

Run Hasura GraphQL engine on Heroku | Hasura GraphQL Docs

**Reply by tatsu38** [reply]
This method can only be used once. This is because https://github.com/nocodb/nocodb-seed-heroku will not be updated.

Therefore, you need to upgrade using Docker directly from the second time onwards.

```
heroku container:login
docker pull nocodb/nocodb:latest
docker tag nocodb/nocodb:latest registry.heroku.com/<Heroku App Name>/web
docker push registry.heroku.com/<Heroku App Name>/web
heroku container:release -a <Heroku App Name> web
```

Nuxt.js, the framework used by NocoDB, has an advanced caching strategy. Be sure to clear your browser's cache after the upgrade.
